src/js/jsx/sections/nodoc/RecentFiles.jsx

```diff
@@ -61,6 +61,7 @@ define(function (require, exports, module) {
         render: function () {
             var recentFilesLimited = this.props.recentFiles.slice(0, MAX_RECENT_FILES),
                 shortenedPaths = pathUtil.getShortestUniquePaths(recentFilesLimited),
+                recentFilesCount = recentFilesLimited.length,
```

Comment:
If `this.props.recentFiles` and hence `recentFilesLimited` are `Immutable.List`s instead of `Array`s then this will cause a warning in the console because `length` is not the correct property. Instead, you want to use `size`. http://facebook.github.io/immutable-js/docs/#/List/size